test/spec/services/layerHelper.spec.js

```diff
@@ -0,0 +1,98 @@
+goog.require('ngeo.LayerHelper');
+
+describe('ngeo.LayerHelper', function() {
+  var ngeoLayerHelper;
+  var map;
+  var layer;
+  var wmtsSrc = 'http:///wmts/capabilities.xml';
+  var wmtsName = 'org.wmts.layer';
+  var authRequestHandler;
+
+  beforeEach(function() {
+    map = new ol.Map({});
+
+    inject(function($injector) {
+      ngeoLayerHelper = $injector.get('ngeoLayerHelper');
+      $httpBackend = $injector.get('$httpBackend');
+      $httpBackend.when('GET', wmtsSrc).respond();
```

Comment:
My tests on WMTS covers not all my code. To do that, I need to pass a fake, well formed, getCapabilities XML in this 'respond' function... but that's a long string to put there... Some advices ?